financial-templates-lib/GasEstimator.js

```diff
@@ -42,17 +42,17 @@ class GasEstimator {
     }
   };
 
-  _update = async () => {
-    let returnedPrice = await this.getPrice(url);
-    this.lastFastPriceGwei = returnedPrice;
-  };
-
   // Returns the current fast gas price in Wei, converted from the stored Gwei value.
   getCurrentFastPrice = () => {
     return this.lastFastPriceGwei * 1e9;
   };
 
-  getPrice = async url => {
+  _update = async () => {
```

Comment:
Do you think we should replicate the `update` and `forceUpdate` setup/terminology we use in other clients?

Reply:
I think we can add `forceUpdate` if we see a need to do it, but none of the clients knowingly trigger a gasPrice change, unlike how they knowingly trigger an EMP Client state change. 

I think a situation that could potentially call for it is if the client bots detect heightened ETH network fee volatility, they might force an update.

Reply:
Okay, SGTM. We can always add later if we find we need it.